src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  // Rozdzielamy ciąg na linie, usuwamy puste linie i linie zawierające tylko białe znaki
+  const lines = sourceString.split(';').filter(line => line.trim());
+
+  lines.forEach(line => {
+    // Rozdzielamy nazwę właściwości i wartość
+    const [property, value] = line.split(':');
```

Comment:
When splitting the line by ':', there might be cases with leading or trailing whitespaces in property names and values. It's good that you are trimming those, but consider using a more robust method like a regular expression to split and capture trimmed property and value in one step.